src/sharing/state.js

```diff
@@ -65,7 +65,8 @@ export const revokeSharingLink = permissions => ({
 })
 export const receivePaths = paths => ({ type: RECEIVE_PATHS, paths })
 export const matchingInstanceName = (instanceUri = '') => shareMember =>
-  shareMember.instance.toLowerCase() === instanceUri.toLowerCase()
+  shareMember.instance &&
```

Comment:
So a shareMember can not have an instance? What does that really mean?

Reply:
Yeah, when tehy haven't accepted the sharing yet, ie. `pending` status.